lib/catalog/martyrology.ts

```diff
@@ -3167,6 +3421,29 @@ export const Martyrology: { catalog: MartyrologyCatalog } = {
       name: 'Oliver Plunket',
       titles: [Title.Bishop, Title.Martyr],
     },
+    // src:
+    // - mr_fr_1982_ed2_coutances
+    // - https://www.diocese50.fr/les-paroisses-et-les-doyennes/notre-dame-coutances/les-saints-du-diocese/saint-omer-eveque
+    // - https://en.wikipedia.org/wiki/Audomar
+    omer_audomar_bishop: {
+      canonizationLevel: CanonizationLevels.Saint,
+      name: 'Omer',
+      titles: [Title.Bishop],
+      dateOfBirth: 600,
+      dateOfDeath: '670-11-01',
+    },
+    // src:
+    // - mr_fr_1982_ed2_coutances
+    // - https://www.diocese50.fr/les-paroisses-et-les-doyennes/notre-dame-coutances/les-saints-du-diocese/saint-ortaire-abbe
+    // - https://fr.wikipedia.org/wiki/Ortaire_de_Landelles
+    ortaire_de_landelles_abbot: {
```

Comment:
@TobiTenno, I might be wrong, but I think `de` should be replaced by `of`, as `Landelles` is the name of the monastery where he was an abbot. However, I could not find many good sources (they all mostly say [`Ortaire, Abbot of Landelles`](https://www.newadvent.org/cathen/04455b.htm)). I let you decide which form is/sounds better in English.

```suggestion
    ortaire_of_landelles_abbot: {
```

Reply:
I think the article `of` is better than `de`, I just forgot to translate it.